dhis2/dhis2-core
# fix: remove deleted organisation unit from users [DHIS2-8953]

### Summary
When deleting an organisation unit it has to be removed from all 3 of the member collections a user has.

### Automatic Testing
Added a test where a user had the deleted OU in all of the 3 member collections.

### Manual Testing
* create a new OU
* edit a user and select the created OU in all three OU trees (capture, search, data output)
* delete the OU
* deletion should be successful
* check the user, the OU should no longer be shown or selected in the trees
